web/src/components/tasks/business-formation/review/ReviewBusinessSuffixAndStartDate.tsx

```diff
@@ -22,6 +23,15 @@ export const ReviewBusinessSuffixAndStartDate = (): ReactElement => {
     return `${name} ${suffix}`;
   };
 
+  const formattLawResponse = (willPracticeLaw: boolean | undefined): "Yes" | "No" | undefined => {
+    if (willPracticeLaw === true) {
+      return "Yes";
+    } else if (willPracticeLaw === false) {
```

Comment:
pebble - you don't need the else if statement here; the reason for this is that an undefined value will result in a false boolean value, thus, you would never hit line 32. You can return "NO".

Reply:
NVM, I think we want to return a different value for when it's not selected by the user

Reply:
So the way the ReviewLineItem component is written, when a value is passed if that value is undefined it will display the <ReviewNotEntered /> component, but if the content is defined it will display that content. 

So the text will come back as that text and the undefined will render the ReviewNotEntered associated text